src/muya/lib/contentState/enterCtrl.js

```diff
@@ -423,7 +423,7 @@ const enterCtrl = ContentState => {
         // cursor at end of paragraph or at begin of paragraph
         if (type === 'li') {
           if (block.listItemType === 'task') {
-            const { checked } = block.children[0]
+            const checked = false
             newBlock = this.createTaskItemBlock(null, checked)
```

Comment:
Just use `false`?

```diff
-          const { checked } = block.children[0]
-          newBlock = this.createTaskItemBlock(null, checked)
+          newBlock = this.createTaskItemBlock(null, false)
```

Reply:
In fact, the purpose here is to make the code clearer, knowing that the second parameter is `checked`, and whether the task list item is selected.